src/groundcrew/tools.py

```diff
@@ -350,3 +370,298 @@ def __call__(self, user_prompt: str, include_code: bool) -> str:
         prompt += f'{user_prompt}\n\n'
 
         return self.llm(prompt)
+
+
+class CyclomaticComplexityTool:
```

Comment:
`get_complexity` and `summarize` don't use anything from `self`, so they technically could be moved to a utils file or something. I do like keeping them in the class for organization though, so we could add `@staticmethod` to them?

Reply:
Open to discuss this, but going off "vibes", these methods don't feel like static methods to me. E.g., they are not meant for general use outside the tool instance.  `code_utils.cyclomatic_complexity` would be the general purpose function, which is called and manipulated w/in the class.

True that `self` is unused, but the fact that `self` is a required arg to all python class methods is one of my gripes about python in general...  again, open to discuss, but my vote would be to not have them as static methods.

Reply:
@cameronfabbri @jmpounders One solution might be making a private (start with double underscores) static method? The double underscores would indicate not for general use, but static illustrates the lack of dependencies.